### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by evans22j**
In this project, we were told to do the following on the contact form;
-Name field which accepts up to 30 chars.
-Email field that accepts only emails.
-Message field which accepts up to 500 chars.
-All fields have placeholder text.
-All fields have the correct type.
-A user cannot submit the form if one or more of the fields is empty.
-We are collecting data by using the [Formspree service](https://formspree.io/html).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,196 +1,228 @@
...
+              placeholder="Email"
+              class="form"
+              id="email"
+              required
+            />
+            <textarea
+              name="message"
+              id="textarea"
+              placeholder="Write your message here"
+              maxlength="500"
```
Comment (Amrhub) [inline index.html:287]: - [ ] Let's make sure we have `required` on the comments textarea message as we don't want to get emails with an empty message.

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @evans22j  and @kaitcham ,  👋🏻 

Phenomenal Job! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Clean code ✔️ 
Meaningful commit messages ✔️ 
Correct workflow ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @evans22j, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -180,27 +180,48 @@ <h2 class="title">Multi-Post Stories</h2>
...
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding. I’d love to help
-          with it
-        </p>
-        <form
-          action="https://formspree.io/f/mvolakkg"
-          method="POST"
-          class="form">
-          <input type="text" name="name" placeholder="Name" maxlength="30" />
-          <input type="email" name="email" id="" placeholder="Email" />
```
Comment (Achilles-Dev) [inline index.html]: Please it would be appropriate to provide a name for the input id, or better still remove it completely

**Review by Achilles-Dev — CHANGES_REQUESTED** [review]
# Changes required :recycle:.

Please check my comments below and make all required changes.

Submit for re-review when you are done.

Thanks :smile:

**Pull request merged**